src/converters/transactionsConverter.ts

```diff
@@ -28,6 +28,10 @@ export class TransactionsConverter {
             guardian: transaction.guardian ? transaction.guardian : undefined,
             signature: this.toHexOrUndefined(transaction.signature),
             guardianSignature: this.toHexOrUndefined(transaction.guardianSignature),
+            relayer: transaction.relayer ? transaction.relayer : undefined,
+            innerTransactions: transaction.innerTransactions.length
```

Comment:
Later comment: actually, it's good that `toPlainObject()` returns undefined if these new fields are missing :+1: 

Otherwise, downstream, some errors might have occurred. E.g. sending `toPlainObject()` to a gateway etc. (while these new fields - even if empty - aren't supported yet).

Reply:
I am not sure, here maybe it could work but in other places we check the length and make it undefined anyway if there's no inner transaction(e.g. serializing for signing). Is there a reason why we'd make it an empty array instead of undefined?

Reply:
It's good as it is (see the later comment) :+1: 

Generally speaking, reason for empty array instead of undefined: leads to less errors in client code (e.g. better to iterate over empty arrays, than pre-checking if an array is defined, then iterate over it).

But in this context, it's good.